vector/v.perturb/v.perturb.html

```diff
@@ -85,8 +85,3 @@ <h2>AUTHOR</h2>
 <a href="http://www.purdue.edu/">Purdue University</a>
 <p>Random number generators originally written in FORTRAN by Wes Peterson and
 translated to C using <i>f2c</i>.
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 72 unnecessary one empty line.

`<h2>AUTHOR</h2>` should be `<h2>AUTHORS</h2>`.